internal/logging/logging_test.go

```diff
@@ -105,3 +106,176 @@ func TestPackCLILogger(t *testing.T) {
 		})
 	})
 }
+
+func TestColorOuput(t *testing.T) {
```

Comment:
AFAIK, we've standardized to us BDD style testing via the use of `spec`. This should follow that standard unless I'm mistaken.